README.md

```diff
@@ -1,2 +1,37 @@
-# intuit-login-page-clone
-This is a clone of intuit website's login page clone
+# intuit-signup-page-clone
+This is a clone of intuit website's signup page clone
+
+## BUILT WITH
+
+- HTML
+
+- CSS
+
+## Live Demo
+- [Live Demo Link](https://adejam.github.io/intuit-signup-page-clone/index.html)
+
+## Images
+![Youtube video page clone](/images/signup-page.png)
+## Screenshot: 
+![Youtube video page clone](/images/signup-page.png)
+
+
+
+## Author
+### Adeleye Jamiu
+- [@adejam](http://github.com/adejam)
+
+## Contributing
+Contributions, issues and feature requests are welcome!
+
+
+## Show Your Support
+Give a star if you like this project!
+
+
+## Acknowledgments
```

Comment:
[optional]

I think you can also reference the webpage you cloned and Odin project where the assignment is from. :thinking:

Reply:
Thanks for the comment. I'll add the reference to the webpage that was cloned and the Odin project where the assignment is from.